### ISSUE TITLE
Require AA conformance for accessibility

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1454]
Should we consider requiring level AA conformance to meet the accessibility specification to avoid having too much disparity in content?

As it stands, we could end up allowing conformance to multiple versions of WCAG along with either A or AA within any version. AA is the more generally required standard for accessibility.

**Comment by clapierre** [comment]
Our GCA™ program always required AA and we now have 4 certified publishers and 4 conversion vendors all meeting WCAG-AA

So my vote would be to make AA a requirement

**Comment by GeorgeKerscher** [comment]
In the EU, AA is required, so I’ll +1 for AA to be the required level. It also simplifies our work, and encourages this as the worldwide standard.

**Comment by murata2makoto** [comment]
Will already-published EPUB publications be affected by this proposed change?

**Comment by avneeshsingh** [comment]
It is important to know what publishers think about it.
In 2016 we we relaxed the strictness of EPUB accessibility a little because publishing industry was not ready for such a steep curve. 
I have an impression that publishing industry is now OK with level, but keeping my assumptions aside, I would like to invite inputs from the publishers.

**Comment by gregoriopellegrino** [comment]
It is OK for the Fondazione LIA and our member publishers to include this request, but it is difficult to rework already published ebooks.

**Comment by mattgarrish** [comment]
> Will already-published EPUB publications be affected by this proposed change?

No, we'll have to have a new conformance identifier for the new version so old content doesn't have to be updated to conform to the new standard. It will continue to conform to the 1.0 standard.

**Comment by clapierre** [comment]
Why would we need a new conformance identifier? Unless you mean a new "conformance identifier" is a new value of "conformsTo".
 
Wouldn't the conformsTo just point inside the new accessibility specification to either WCAG-A / AA/AAA which would itself point to WCAG 2.1 instead of 2.0

By pointing to the new Accessibility standard @ W3C instead of the old IDPF standard it does three things.
1. which version of the Accessibility Standard this complies with (1.0 / 2.0 / ...)
2. what Level of WCAG this complies with (A/AA/AAA)
3. what version of WCAG this complies with (2.0/2.1/...)

**Comment by mattgarrish** [comment]
> Unless you mean a new "conformance identifier" is a new value of "conformsTo".

Yes, the old identifier URL references the IDPF specification, which is 1.0. It's fine for content that conformed to 1.0 to continue to conform to 1.0. We can't expect authors to reevaluate all their content every time we update.

We should probably add an advisory note that when you re-release old content it should be rechecked to the newest standard, though.

How we update the conformance identifier is a discussion for issue #1455. If we allow conformance to either 2.0 or 2.1, and either A or AA, then reporting gets complicated.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-01-14-epub-a11y#section2) on 2021-01-14

List of resolutions:

- ***[Resolution No. 1](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-01-14-epub-a11y#resolution1):  maintain recommendation of AA, but provide explanations why levels are recommended..***

<details><summary><i>View the transcript</i></summary>

### 2. Make WCAG AA a requirement for EPUB Accessibility 1.1 

_See github issue [#1454](https://github.com/w3c/epub-specs/issues/1454)._




**Matt Garrish:** this is another question of whether we set a floor, and what floor, but for WCAG levels (A, AA, etc.).  
… should we set a floor at AA, which is what most legislation seems to be heading towards?.  

**Charles LaPierre:** for Global Certified Accessible, the floor is AA right now.  
… we were concerned that publishers wouldn't get on board.  
… but we've found support for the AA standard.  
… in practice.  

**George Kerscher:** i think that the real hurdles for publishers is in level A anyway, so moving to AA is not that much more difficult, in my mind.  

**Cristina Mussinelli:** agree with respect to current and future publications, but there is still the issue of existing publications.  

**Charles LaPierre:** the language of parts is in the AA level, so books written in multiple languages are a lot of work for publishers wishing for AA level.  
… right now, for GCA, publishers who do these types of books are just complying with level A.  

**Avneesh Singh:** level A is also fairly strict for fixed layouts.  

**Matt Garrish:** with respect to AAA, it is very theoretical. Nothing really uses it right now.  
… in terms of meeting level A, don't forget that it is always an option for publishers to state that they meet Level A without strictly adhering to a11y spec.  

**Avneesh Singh:** what we've heard now is mostly from reps in US and Italy etc. but a11y compliance is way behind in other parts of the world.  
… would setting a floor at AA, would this be unrealistic for publishing in other parts of the world?.  

**George Kerscher:** About the language switching, we will need to put out some best practices about this. Just because there is a reference to something in an english book that is from a foreign language, there doesn't need to be a lang tag around it..  
… there are lots of borrowed terms that are generally accepted as being english.  

**Tzviya Siegman:** when we write accessibility statements for websites, we state that we are "striving" for AA compliance, because we know that there could be things missing.  
… similarly for publishers, even those who have been certified, it is still possible for something to be missing from one of the titles.  
… if a customer identifies an issue, we will go back and remediate it, but that doesn't mean that we should hold ourselves to a lower standard.  

**George Kerscher:** +1. We'd expect to get objections if our spec required perfection.  

> *Bill Kasdorf:* +1 to George.

**Avneesh Singh:** “EPUB Publications _MUST_ meet WCAG 2.0 Level A to be conformant with this document, but it is _RECOMMENDED_ that they meet Level AA”.  
… this is what it says currently.  
… does it require change?.  

**Cristina Mussinelli:** maybe it could say "highly" recommended, or "strongly" recommended? Along with greater explanation of why AA is better than A.  

**Bill Kasdorf:** i think it would be stronger to say "SHOULD" rather than "RECOMMENDED".  

**Matt Garrish:** recommended and should are equivalent, normatively speaking.  
… but do we want to be the authority telling people what to do? Or should we be more accommodative?.  
… but maybe we should leave floor setting to legislation, while using the spec to encourage publishers to move to higher levels?.  
… and just clearly setting out why it is important to strive for higher levels of a11y compliance.  

> *Charles LaPierre:* +1 I am fine with that as well..

**Cristina Mussinelli:** we should explain why we encourage one level over another.  

> **Proposed resolution: maintain recommendation of AA, but provide explanations why levels are recommended..** *(Avneesh Singh)*

> *Matt Garrish:* +1.

**Avneesh Singh:** leaving it to MattG to come up with the final language.  

> *Tzviya Siegman:* +1.

> *Juliette McShane:* +1.

> *Charles LaPierre:* +1.

> *Bill Kasdorf:* +1.

> *Matthew Chan:* +1.

> *Cristina Mussinelli:* +1.

> *George Kerscher:* +1.

> *Ben Schroeter:* +1.

> ***Resolution #1: maintain recommendation of AA, but provide explanations why levels are recommended..***


</details>
